ngx-fudis/projects/ngx-fudis/src/lib/components/form/guidance/guidance.component.ts

```diff
@@ -121,6 +126,7 @@ export class GuidanceComponent implements OnInit {
 
   ngOnInit(): void {
     if (this.maxLength) {
+      this._maxLengthWidth = this.maxLength > 100 ? 'lg' : this.maxLength > 10 ? 'md' : 'sm';
```

Comment:
This needs to be refactored away from ngOnInit to ngOnChanges, so if for some reason maxLength value changes the values must be calculated again.